src/QuadForm/Misc.jl

```diff
@@ -879,7 +879,7 @@ function _find_quaternion_algebra(b, P, I)
     end
     M = vcat(M, v)
     push!(elts, f(L[i])) # cache
-    fl, w = can_solve_with_solution(M, target, side = :left)
+    fl, _ = can_solve_with_solution(M, target, side = :left)
     if fl
```

Comment:
```suggestion
    if can_solve(M, target, side = :left)
```
no need to compute the solution and then discard it